platform/entanglement/src/capabilities/LocationCapability.js

```diff
@@ -0,0 +1,77 @@
+/*global define */
+
+define(
+    function () {
+        "use strict";
+
```

Comment:
It would be good to have some class-level JSDoc here ("Implements the `location` capability which...")